docs/learning-powershell/PowerShellConfigurationSettings.md

```diff
@@ -0,0 +1,100 @@
+# `PowerShell Core` configuration settings
+
+`PowerShell Core` is configured using the following schemes:
+
+- On Windows - Group Policy Objects (GPO), Group Policy Preferences (GPP) and configuration files
+- On Unix - configuration files.
+
+**Caution!** The configuration schemes differ from `PowerShell Core` _profile_ files.
+
+Configuration schemes allow to customize `PowerShell Core` in the most flexible way:
+
+- Enterprise administrators can use GPO, GPP and computer-wide confuguration files to apply approved configuration settings and mandatory security settings in a centralized manner. The same settings can be applied at user or application levels.
+- Developers and consumers can use user and application level files.
+
+`PowerShell Core` settings are grouped into policies and options. Options are normal configuration settings. Policies is high priority and overlap options.
+
+## Priority of applying settings
+
+Because a configuration setting can be in several schemes, the setting wins according to the priority of its scheme.
+
+### Priorities for Policies in descending order
+
+Scheme | Windows | Unix
+-| - | -
+GPO -> Computer Policy | HKLM\Software\Policies\Microsoft\Windows\PowerShellCore | /etc/powershell.config.json
```

Comment:
The PowerShell Core GPO registry keys are at `Software\Policies\Microsoft\PowerShellCore`.